draft-ietf-httpbis-messaging-latest.xml

```diff
@@ -405,7 +405,8 @@
 <t>
    Intermediaries that process HTTP messages (i.e., all intermediaries
    other than those acting as tunnels) &MUST; send their own HTTP-version
-   in forwarded messages.  In other words, they are not allowed to blindly
+   in forwarded messages, unless it is purposefully downgraded as a workaround
```

Comment:
```suggestion
   in forwarded messages, unless a request is purposefully downgraded as a workaround
```

This only applies to requests, right?  The expectation is that a server is able to produce a lower-version in its response.

Reply:
While Willie talked about the request side, it might be relevant on the response side too.

Reply:
both ways